example_scripts/download_vhi.py

```diff
@@ -0,0 +1,22 @@
+from ftplib import FTP
```

Comment:
@gabrieltseng this is a test file that shows how the FTP server requests work. It downloads a single file.